src/lib/store/remote/api.js

```diff
@@ -140,10 +140,20 @@ exports.getSinceNr = function(state) {
 // changes since the beginning, but this behavior might be adjusted,
 // e.g for a filtered bootstrap.
 //
+// As several firewalls and anti virus software strip out Accept-Encoding
+// request headers, ~15% of users end up with much bigger download sizes
+// than needed. So we created a workaround to force gzip via url param,
+// and fallback to normal request if it fails:
+// https://github.com/hoodiehq/hoodie.js/issues/423
+//
 exports.bootstrap = function(state) {
   state.isBootstrapping = true;
   state.remote.trigger('bootstrap:start');
-  return state.remote.pull()
+  return state.remote.pull({forceGzip: true})
```

Comment:
I think `forceGzip: true` is more clear in terms what it does, don't you think? And I don't see a scenario where we would need something like `gzip: false`